components/dnd_custom_time_picker_modal/dnd_custom_time_picker_modal.tsx

```diff
@@ -162,7 +162,9 @@ export default class DndCustomTimePicker extends React.PureComponent<Props, Stat
                 <div className='DndModal__content'>
                     <div>
                         <div className='DndModal__input DndModal__input--no-border'>
-                            <div className='DndModal__input__label'>{'Date'}</div>
+                            <div className='DndModal__input__label'>
+                                {localizeMessage('dnd_custom_time_picker_modal.date','Date')}
+                            </div>
```

Comment:
Could you please use FormattedMessage here as well, so we stay consistent? We could also remove `localizeMessage` from imports, since this is the last time it gets used.